hvpy/io.py

```diff
@@ -55,3 +55,10 @@ def url(self) -> str:
         Final API endpoint URL.
         """
         return BASE_URL + self.__class__.__name__[:-15] + "/"
+
+    @validator("date", "startTime", "endTime", check_fields=False)
+    def convert_date_to_isoformat(cls, v) -> str:
+        """
+        Converts the date from a datetime object to a string in the ISO format.
+        """
+        return v.isoformat() + "Z"
```

Comment:
I don't think this will work:

```
E   pydantic.error_wrappers.ValidationError: 2 validation errors for getJPXInputParameters
E   startTime
E     HvpyParameters.convert_date_to_isoformat() missing 1 required positional argument: 'v' (type=type_error)
E   endTime
E     HvpyParameters.convert_date_to_isoformat() missing 1 required positional argument: 'v' (type=type_error)
```